### DISCUSSION TITLE
How to simulate the startup of Openfast

### DISCUSSION THREAD

**Discussion by 17638031880** [OpenFAST/openfast#2068] [General]
Which parameters to modify？

**Comment by jjonkman** [comment]
Dear @17638031880,

Can you clarify your question?  What kind of "startup" are you trying to simulate in OpenFAST?

FYI: General guidance on simulating various start-up and shut-down events is provided in the old FAST v6 User's Guide: https://openfast.readthedocs.io/en/main/_downloads/d8bd014121d6505cb25cf49bee5eaa80/Old_FAST6_UsersGuide.pdf.

Best regards,
